fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleType.java

```diff
@@ -38,6 +38,7 @@ public enum RuleType {
     PROJECT_TO_GLOBAL_AGGREGATE(RuleTypeClass.REWRITE),
 
     // rewrite rules
+    AGGREGATE_DISASSEMBLE(RuleTypeClass.REWRITE),
```

Comment:
It seems that it is now split into two-stage aggregation by default? Should a todo be added, indicating that whether to apply the rule or not will be determined according to the cost situation in the future.

Reply:
added TODO